docs/source/model_design/risk_factors.rst

```diff
@@ -195,7 +195,118 @@ relationships directionally between a cause/exposure and effect/outcome.
+
+In **observational studies**, however, it is rarely possible for individuals to be randomly assigned to an exposure. Often, individuals who share a particular risk factor have other characteristics in common that influence their risk of disease. Individuals who do not share this particular risk factor may also differ in other important ways that influence their risk of disease. So we cannot be sure that those with and without the risk factor of interest (exposed and unexposed individuals) are similar, or comparable, with respect to all other relevant factors. This makes it difficult to determine if the association we observe between disease and our risk factor of interest is real, or whether it is influenced by other factors.
+
+**An example**:
+Imagine that you are interested in knowing whether smoking (our risk factor of interest) influences the risk of coronary heart disease (CHD) in men aged 18 to 64 years. You conduct a cohort study with an exposed group of male smokers in this age group from the general population, and compare their risk of CHD after a number of years to that of an unexposed group of non-smoking males in the same age group. You would expect to find a higher incidence of CHD among smokers. However, males who smoke are also likely to have a higher alcohol intake, which also increases the risk of CHD. Thus, the effect that you observe for the smoking-CHD association is **mixed up** with the effect of the association between alcohol intake and CHD. Without taking account of differences in alcohol intake between smokers and non-smokers, the magnitude of the smoking-CHD association estimated from the study may be higher than the **true** value, leading to incorrect conclusions.
+
+The criteria for alcohol to be a confounding factor for the smoking-CHD relationship is met because:
+
+  1. Alcohol intake is a risk factor for CHD in non-smokers. In other words the relationship between alcohol and CHD is not dependent on smoking status.  
+  2. Smokers generally have a higher alcohol intake (in most populations). 
+  3. Finally, alcohol intake is not on the causal pathway between smoking and CHD (smoking does not in itself cause people to drink more alcohol). 
+
+.. image:: smoking_triangle.svg
+
+Note that the dotted line between smoking and alcohol indicates that we do not expect a 'causal' link between the smoking and alcohol. Rather, it indicates that the two are associated in the population.
+
+**How to identify confounding?**
```

Comment:
I wonder if it would be valuable to move this to a general epi section like the study design stuff (since stratification is really valuable for effect modification also). Doesn't need to be done now necessarily, but we can maybe talk about it at our next docs session.

Reply:
ok sounds good :)  I've addressed all the comments. I'm gona merge it so we have it up. We can go through it and see how we can address some of the more structural things like have a general epi section.